HI, Could you explain me what is wrong ? 

Run python3 generator.py
Traceback (most recent call last):
  File "/home/runner/work/ton-assets/ton-assets/generator.py", line 1[5](https://github.com/tonkeeper/ton-assets/actions/runs/9161159020/job/25550331830?pr=548#step:5:6)2, in <module>
    main()
  File "/home/runner/work/ton-assets/ton-assets/generator.py", line 95, in main
    jettons = merge_jettons()
  File "/home/runner/work/ton-assets/ton-assets/generator.py", line [6](https://github.com/tonkeeper/ton-assets/actions/runs/9161159020/job/25550331830?pr=548#step:5:7)5, in merge_jettons
    raise Exception("don't use cache.tonapi.io as image source in %v", j.get("name"))
Exception: ("don't use cache.tonapi.io as image source in %v", 'GPU — M2M coin')
Error: Process completed with exit code 1.


here i can see ""don't use cache.tonapi.io as image source in %v"" but i don't use cache.tonapi.io link.

--- a/jettons/ODER.yaml
+++ b/jettons/ODER.yaml
@@ -2,8 +2,7 @@ name: ECNATSISER GOD
 description: ODER represents the upside version of REDO.
 image: "https://raw.githubusercontent.com/ODER12/Logo/main/LogoOder.png"
 address: EQAyEjcjLF24H2TTrey1Yed3uRs18xbqkFCU3uzphHHr8n3y
-symbol: ODER
-max_supply: 10000000 
+symbol: ODER 
 websites:
   - "https://ecnatsisergod.com/"
 social:

